src/modules/did/didSyncStatus.entity.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { DIDDocumentEntity } from './did.entity';
+import { DidSyncStatus } from './did.types';
+
+@Entity()
+export class DidSyncStatusEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @OneToOne(() => DIDDocumentEntity, (document) => document.id)
+  @JoinColumn()
+  document: string;
```

Comment:
It seems wrong to me that this property is `string` type. I think that typically in a one-to-one join column, the join property has the same type as the joined entity.
See the `Profile` example at the start of the TypeORM documentation: https://orkhan.gitbook.io/typeorm/docs/one-to-one-relations
```suggestion
  @OneToOne(() => DIDDocumentEntity, (document) => document.id)
  @JoinColumn()
  document: DIDDocumentEntity;
```

Reply:
Yes, I overlooked that